### ISSUE TITLE
config values can sometimes be globals, not something that needs to be set in configuration

### ISSUE THREAD

**Issue by scripting** [scripting/feedlandInstall#55]
BTW, there was some confusion as to whether `config.flFeedsHaveIds` needs to be set by the installation. It doesn't and it shouldn't. 

config values can also be globals, set by the software itself, and this is one of those values.

When feedland server boots up, it checks to see if the feeds table has an ID column, and if it does, it sets config.flFeedsHaveIds true, otherwise false, overwriting whatever the installer put there. It probably shouldn't be initialized to false in the declaration of config, because that is itself confusing.

At this time there is only one other value like this, `config.mysqlVersion`. Both values are set in the [startup code](https://github.com/scripting/feedland/blob/main/feedland.js#L1516) in feedland.js.

So to summarize, sometimes `config.x` is a global not really a config value.
